lib/chef/provider/aws_route_table.rb

```diff
@@ -61,8 +71,9 @@ def update_routes(vpc, route_table, ignore_route_targets = [])
     current_routes = {}
     route_table.routes.each do |route|
       # Ignore the automatic local route
-      next if route.target.id == 'local'
-      next if ignore_route_targets.find { |target| route.target.id.match(/#{target}/) }
+      route_target = route.gateway_id || route.instance_id || route.network_interface_id || route.vpc_peering_connection_id
```

Comment:
So route no longer has `route.target`.  Are we sure this is the full list of available targets?  Will this need to be updated in the future if new targets get added?